src/swap-routers/WethToWstEthSwapRouter.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.13;
+
+import {BaseSwapRouter} from "./BaseSwapRouter.sol";
+
+import {Address} from "openzeppelin-contracts/utils/Address.sol";
+import {SafeTransferLib} from "solmate/utils/SafeTransferLib.sol";
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import {Constants as C} from "../lib/Constants.sol";
+import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
+import {IwstETH} from "../interfaces/lido/IwstETH.sol";
+import {WETH} from "solmate/tokens/WETH.sol";
+import {ICurvePool} from "../interfaces/curve/ICurvePool.sol";
+import {ILido} from "../interfaces/lido/ILido.sol";
+
+contract WethToWstEthSwapRouter is BaseSwapRouter {
```

Comment:
I'm a bit confused with these two swap router contracts. why do you need 2 contracts and not one with 2 separate functions for different swaps? when executing the swaps you anyways need to know which one to call so you can call a specific method instead, right? maybe I'm missing something